scipy/optimize/_linprog_highs.py

```diff
@@ -344,7 +347,7 @@ def _linprog_highs(lp, solver, time_limit=None, presolve=True,
         'sense': HIGHS_OBJECTIVE_SENSE_MINIMIZE,
         'solver': solver,
         'time_limit': time_limit,
-        'highs_debug_level': MESSAGE_LEVEL_MINIMAL * disp,
+        'highs_debug_level': MESSAGE_LEVEL_NONE * disp,
```

Comment:
This should probably be turned back up to minimal when possible

Reply:
Why did it need to be changed for now?

Reply:
This in particular is a bit more confusing than it has to be because I was trying to reduce the diffs. The HiGHS logging and debug mechanisms have been completely overhauled since we last updated.  This means some of the previous Enums no longer exist and all have been renamed using a different naming convention.  The `MESSAGE_LEVEL` Enum has been removed entirely, and I mapped `MESSAGE_LEVEL_MINIMAL` to the new `HighsDebugLevel::kHighsDebugLevelCheap`.  This `HighsDebugLevel` induces extra checks on the solution that produce "false negatives" for MIP solutions and report them as having errors when there are none (see https://github.com/ERGO-Code/HiGHS/issues/633).  Until the upstream issue is resolved (or we find a different `HighsDebugLevel` to use), I'm leaving it at no debug level to make sure those flase-negative checks are skippeed

Reply:
Generally, you shouldn't be running HiGHS with anything other than HighsDebugLevel::kHighsDebugLevelNone. These debug levels are things for us to set when developing, or for me to suggest users set when experiencing anomalous behaviour.

Reply:
@jajhall Good to know -- is there something analogous to "logging level" we can have users set to output console information or is it simply enough to set `log_to_console = true`?

Reply:
`log_to_console` is true by defaut, and `log_file` is "" by default, so no log file is created. Hence, by default, the appearance of logging output (to console) is controlled by the setting `output_flag`. By default it is true, so logging to console appears. Equivalently, you can keep `output_flag` and control the appearance of logging output to console using `log_to_console`.

As I understand, the operation of these flags is identical to that of Gurobi

Reply:
Solved in gh-15217